backend/src/main/java/de/laura/project/service/TriviaService.java

```diff
@@ -82,9 +82,13 @@ public void addIDtoQuestionSet(List<TriviaQuestionSet> triviaQuestionSetList) {
         };
     }
 
-    public TriviaQuestionSetWithoutAnswer getSingleQuestion(int id) {
+    public TriviaQuestionSetWithoutCorrectAnswer getSingleQuestion(int id) {
         return tempTriviaQuestionDB.getSingleQuestion(id);
     }
+
+    public boolean checkAnswer(int questionID, String selectedAnswer) {
+        return tempTriviaQuestionDB.checkAnswer(questionID, selectedAnswer);
```

Comment:
why is your DB named `temp`? This indicates a WIP solution. Is this supposed to be substituted later on?